### DISCUSSION TITLE
Titles for Chromie points

### DISCUSSION THREAD

**Discussion by krutoj87** [chromiecraft/chromiecraft#7624] [Suggestions]
As an incentive for people to support CC you could add some custom titles as vanity items for ChromiePoints.
Maybe the community could suggest titles in some kind of contest and you would pick some of them to be buyable for CP later on. The winners of the contest could receive the title they suggested and if you choose also something else. 

Also you could add an option to suggest titles for CP later on. If players suggest a title somebody from the staff could Review it and if it doesn't break any rules and fits to the game, it could be added to the list of buyable titles and the player would also receive this title.
If players suggest titles that are against the rules, they wouldn't be added and the player also shouldn't be refunded the CP.
If you don't want players to be able to buy titles, you could also just make contests periodically, so the catalogue of titles grows.

Another idea, that would probably be much harder to implement, but might be interesting for hardcore players are dynamic titles depending on how many deaths you've had. 
Something like "The 3rd", where the number would be the number of characters you've created with the same name.
For this to work it would need some prerequisites.
1. Players would need to register a Name that would be saved and only characters with this Name would be counted. For example I register "Refusetodie" so whenever the character with this Name dies and I create a new one, the number would be incrmented. If I create "Refusetolive" it wouldn't be incremented and this char also wouldn't receive the title. 
2. There would have to be a way for players to register character names, so only that one Account can create characters with that name. I think this already exists with the subscription.
3. There would need to be a database that keeps track of registered names, the accounts they belong to, the number of chars that have been created with this name and a flag of the service is enabled for that name. 
4. Since it's an ongoing service it should probably be tied to the CC subscription. Once the subscription runs out the name you registered won't be tracked anymore. It might still remain in the database for a set period, if you decide to continue that Service later on. 
5. The title should only be awarded to players when they activate the hardcore mode. In order to have a max level char with the title "The 1st" I would need to register a name and level a hardcore character to max level. It's like a badge for completing the hardcore challenge. 

So basically if I want a character on the server with this title, I subscribe to CC. I use some of the CP to register the name I'm going to use. Then I create a character with that name and once I activate hardcore mode with that char, I receive the title "The 1st". If I die, delete the char and create a new char, I get the title "The 2nd" once I activate hardcore mode again. If I finish the the hardcore challenge now, I will have a max level character, that is a regular character now, but has the title "The 2nd".
If I die again before finishing and I make a new character again with the same name, but level it to max without activating the hardcore challenge, I will not receive the title.
Now I could either delete or rename the character to create a new character with this name in order to get the title for this name again. Or I spend some more CP while still being subscribed, to register another name. But when I create a char with that other name, it will be "The 1st" again.

It's a lot of work and I don't know if there would be enough players to even use this feature, for it to be worth it. But maybe you could make a poll before implementing it, to see if players are even interested in this feature. 
Also it's just an addition that could be added much later. The main suggestion are the titles players can buy for CP.

**Comment by krutoj87** [comment]
Ok another idea I have to add. You could also add titles for people who are subscribed to CC for X period of time. So for example people who are subscribed for 1 year receive the title "Supporter" and people who are subscribed for 2 years receive the title "Loyal Supporter" or something like this.

**Comment by Nyeriah** [comment]
Hello, 

thanks for your suggestion, however we can’t add custom titles, they require client modifications

**Reply by krutoj87** [reply]
Well that's too bad, but it is what it is.
